tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md

```diff
@@ -143,5 +143,16 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 
 - [apiGraph/addUserToGroup.feature:420](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L420)
 
+### [500 error if viewer tries to lock file](https://github.com/owncloud/ocis/issues/7600)
+
+- [apiSpaces/lockFiles.feature:144](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/lockFiles.feature#L144)
+- [apiSpaces/lockFiles.feature:207](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/lockFiles.feature#L207)
```

Comment:
This issue has already been closed https://github.com/owncloud/ocis/issues/7600 so might not be the correct issue to tag tests on 
cc @saw-jan

Reply:
backport of another PR is needed